proto/terra/feeshare/v1beta1/genesis.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+package terra.feeshare.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "terra/feeshare/v1beta1/feeshare.proto";
+
+option go_package = "github.com/classic-terra/core/x/feeshare/types";
+
+// GenesisState defines the module's genesis state.
+message GenesisState {
+  // params are the feeshare module parameters
+  Params params = 1 [(gogoproto.nullable) = false];
+  // FeeShare is a slice of active registered contracts for fee distribution
+  repeated FeeShare fee_share = 2 [(gogoproto.nullable) = false];
+}
+
+// Params defines the feeshare module params
+message Params {
+  // enable_feeshare defines a parameter to enable the feeshare module
```

Comment:
Not sure if its a big deal that your not manually implementing the proto stringer method in your Keeper Params?

e.g: https://github.com/classic-terra/core/blob/649f7c4348bc437c6b26bbd9114febef84c7dd3a/x/market/types/params.go#L48

Reply:
Tests worked. Not a big deal, probably a better pattern long term :)